HeterogeneousCore/SonicTriton/test/tritonTest_cfg.py

```diff
@@ -50,6 +58,16 @@
 }
 
 defaultClient = applyClientOptions(getDefaultClientPSet().clone(), options)
+keepMsgs = []
+if options.verbose or options.verboseDiscovery:
+    keepMsgs.append('TritonDiscovery')
+if options.verbose or options.verboseClient:
+    keepMsgs.append('TritonClient')
+if options.verbose or options.verboseService:
+    keepMsgs.append('TritonService')
+if options.verbose:
+    # ensure RetryActionDiffServer messages are not suppressed if emitted
+    keepMsgs.append('RetryActionDiffServer')
```

Comment:
incorrect rebase. these lines are already contained in https://github.com/cms-sw/cmssw/blob/79b590e4ef3ac7018721083e171b90f1d6b13e8f/HeterogeneousCore/SonicTriton/python/customize.py#L124 which is called during https://github.com/cms-sw/cmssw/blob/79b590e4ef3ac7018721083e171b90f1d6b13e8f/HeterogeneousCore/SonicTriton/python/customize.py#L53
remove these lines